### DISCUSSION TITLE
Test Faucet Issue

### DISCUSSION THREAD

**Discussion by gyanshupathak** [Cyfrin/foundry-full-course-cu#2529] [General]
I getting an issue that  i must have 0.01 eth in my mainnet before i can use any of the faucets available in the repo!
How to get that , i just created an metamask account .

**Comment by EngrPips** [comment]
Try this faucet https://faucets.chain.link/sepolia it doesn't ask for `0.01eth` balance.

**Reply by JonasAllenCodes** [reply]
Wish I seen this sooner, I came here to say the docs and/or video should mention this with an update. I notice the video uses the link you mentioned but the docs now no longer show this link. Kinda frustrated by this but it costed me less then $10 to figure this out. Thank you for @EngrPips for the mention of the original link. For some reason I didn't think to just copy the link out of the video.

**Reply by JonasAllenCodes** [reply]
So it looks like further in the course they explain you just need to wait about 20 mins and it will work with out the 0.01eth balance.

**Reply by EngrPips** [reply]
> Wish I seen this sooner, I came here to say the docs and/or video should mention this with an update. I notice the video uses the link you mentioned but the docs now no longer show this link. Kinda frustrated by this but it costed me less then $10 to figure this out. Thank you for @EngrPips for the mention of the original link. For some reason I didn't think to just copy the link out of the video.

How do you mean it cost you less than $10?

**Reply by JonasAllenCodes** [reply]
I went and bought the ETH needed on the mainnet instead of waiting or going to the other link that is not in the docs. I should of came here first though. I really do appreciate this course. I really wish this was around and came here the first time I got into Crypto/Blockchain a few years back. I am deciding to get back into blockchain and development. I just finished the Blockchain Basics course. I am looking forward to moving onto the Solidity 101 course.

**Reply by EngrPips** [reply]
Oh, I see, Sorry you purchased that ether. It's yours still, lol. Yeah, the content of the course is second to none that I know of as of now, and Cyfrin has a lot coming to stick around. If you ever need help while taking the course always reach out on here or Discord https://discord.gg/5q7AmaUq . I wish you the best.

**Reply by JonasAllenCodes** [reply]
Thank you, I created the issue and submitted a pull request to suggest an edit to the README.md. I hope that is okay, I just wanted to suggest an edit to avoid further confusions for future students. Also it is my first pull request, I tried following your contributing guidelines. I will checkout the discord.

**Reply by EngrPips** [reply]
Awesome. You have done well, sir. Thanks for contributing to making the course better. Catch you on Discord, sir.
